### PR TITLE
Remove information about filters object which is no longer required i…

### PR DESCRIPTION
**Pull request by kacperf531**
…n v3.3 when providing a page ID.
FB: https://developers.labs.livechat.com/docs/feature/Remove-filters-info-in-3-3-for-pagination/

## 📓 Description
In v3.3, filters requested on first page are remembered when paginating. Providing `filters` object in the request with `page_id` would result in validation error. note: I'm not sure if this should be included in the changelog, as this has been in place for a while, probably someone just overseen this detail.

## ✅ Checklist (for API docs)

- Changelog
- API versions
- Web & RTM
- **Table:** Available methods/webhooks/pushes
- **Table:** Specifics

For more guidelines, see [Updating API docs](https://livechatinc.atlassian.net/wiki/spaces/PAT/pages/761529400/Updating+API+docs).

### Content

- Proofreading/content fixes

### CODE REVIEW

File: content/messaging/agent-chat-api/v3.3/index.mdx [hunk 1]
```diff
@@ -511,7 +511,7 @@ It returns [summaries](#chat-summaries) of the chats an Agent has access to.
 
 | Parameter                                             | Required | Type     | Notes                                                                                       |
 | ----------------------------------------------------- | -------- | -------- | ------------------------------------------------------------------------------------------- |
-| `filters`                                             | No       | `object` | Mustn't change between requests for subsequent pages. Otherwise, the behavior is undefined. |
+| `filters`                                             | No       | `object` |                                                                                             |
```
Comment (ale-kacperczyk) [inline content/messaging/agent-chat-api/v3.3/index.mdx:514]:
```suggestion
| `filters`                                             | No       | `object` | When paginating, `filters` provided in the first request are remembered and automatically used for the subsequent requests. Providing a new `filters` object will result in a `validation` error.  To reset the filters, start paginating with a new set of filters.|
```

Comment (ale-kacperczyk) [reply content/messaging/agent-chat-api/v3.3/index.mdx:514]: WDYT, @kacperf531?

Comment (kacperf531) [reply content/messaging/agent-chat-api/v3.3/index.mdx]: sure, commited your suggestion, sorry for the delay :(

File: content/messaging/agent-chat-api/v3.3/rtm-reference/index.mdx [hunk 2]
```diff
@@ -558,7 +558,7 @@ It returns [summaries](#chat-summaries) of the chats an Agent has access to.
 
 | Request object                                        | Required | Type     | Notes                                                                                       |
 | ----------------------------------------------------- | -------- | -------- | ------------------------------------------------------------------------------------------- |
-| `filters`                                             | No       | `object` | Mustn't change between requests for subsequent pages. Otherwise, the behavior is undefined. |
+| `filters`                                             | No       | `object` |                                                                                             |
```
Comment (ale-kacperczyk) [inline content/messaging/agent-chat-api/v3.3/rtm-reference/index.mdx:561]: I would add the same note as in Web reference and ask someone from the API team to review it and let us know if it's clear.

Comment (kacperf531) [reply content/messaging/agent-chat-api/v3.3/rtm-reference/index.mdx]: updated in f55e6c166737c4a8049546b77dd370d6a53e97d2

**Review by ale-kacperczyk — APPROVED** [review]

**Review by janowsiany — APPROVED** [review]

**Pull request merged**